README.md

```diff
@@ -1 +1,166 @@
 # JS Adagrams
+## At a Glance
+- Individual, [stage 1](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md#stage-1) project
+- Due before class, **DATE HERE**
+
+## Introduction
+In this project you will build the same Adagrams project from the Ruby curriculum using JavaScript running on Node.js in your terminal. Please review [the original project](https://github.com/AdaGold/adagrams#readme) for an explanation of the Adagrams game design.
+
+A test suite and sample game project have been provided so that you can practice TDD and verify your implementation is correct. A completed project will pass all of the provided tests and have coverage of at least 90%.
+
+## Learning Goals
+
+- Write JavaScript code using functions
+- Practice TDD with JavaScript and the Jest testing framework
+
+## Getting Started
+After forking and cloning this repo you should `cd` to the project directory and run:
+
+```bash
+$ npm install
+```
+
+Similar to `bundle install` in Ruby projects, this makes the npm package manager download and install any dependencies for the project (such as Jest).
+
+### Tests
+We have provided unit tests for you to run. A complete project will pass all provided tests.
+
+To run the tests, in the command line, navigate to the project root and then run:
+
+```bash
+$ npm test
+```
+
+After your tests have run there is a short table outputting the coverage summary for each file. You can view complete coverage details by running:
+
+```bash
+$ npm run coverage
+```
+
+This is shorthand for the command `open coverage/lcov-report/index.html` and will open the coverage report in your default web browser.
+
+### Driver Code
+We have provided some driver code for your Adagrams game in the files `wave-1-game.rb`, `wave-2-game.rb`, `wave-3-game.rb`, and `wave-4-game.rb`. Running `$ ruby wave-1-game.rb` will begin a command-line game that uses your Adagrams code. The boilerplate code will break the first time you run it: working through the waves specified below should create a running version of the game. **Implementing code to make this game run is not a substitute for making the tests pass**. It is simply there for you and your pair to reference how the Game may run during each wave, to have better perspective of what your program can do, and to get some practice reading other peoples' code. We fully expect you to create the specified methods to specification and making the tests pass.
+
+### Project Structure
+This repository has a baseline structure for the project which includes several files. You will only need to modify one of them:
+
+File | Description
+--- | ---
+src/adagrams.js | This is the project code, you implementation should be written here as functions within the `Adagrams` object.
```

Comment:
you implementation -> your implementation

Reply:
Also, do we want it such that all of the functions are within an Adagrams object?

Reply:
I'm looking at it more as equivalent to a Ruby module rather than a class... my implementation at least didn't require any use of `this`, for example.

Reply:
I should definitely clarify that somewhere, or make sure it lines up properly with how we're teaching them in the lecture materials!